lib/car.rb

```diff
@@ -0,0 +1,21 @@
+class Car
+  attr_reader :make_model, :monthly_payment, :loan_length
+  def initialize(make_model, monthly_payment, loan_length)
+    @make_model = make_model.split(" ")
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+  end
+
+  def total_cost
+    @monthly_payment * @loan_length
+  end
+
+  def make
+    @make_model[0]
+  end
+
+  def model
+    @make_model[1]
+  end
+
```

Comment:
Missing the `color` and `paint` methods from the interaction pattern's Iteration 1 (e.g. `car.color`, then `car.paint!(arg)`),